### PR TITLE
Added Glassmorphism CSS Generator

### PR DESCRIPTION
**Pull request by mohitbhusal11**
## Changes proposed
Added new website (Glassmorphism CSS Generator) in Frontend UI-generator ssection

Closes : #985

### CODE REVIEW

**Review by rupali-codes — DISMISSED** [review]
lgtm, thanks

**Comment by rupali-codes** [comment]
@ujjawaltyagii could you please review it? :)

File: database/frontend/ui-generators.json [hunk 1]
```diff
@@ -124,5 +124,12 @@
     "url": "https://superdesigner.co/",
     "category": "frontend",
     "subcategory": "ui-generators"
+  },
+  {
+    "name": "Glassmorphism CSS Generator",
+    "description": "Glassmorphism is a new design trend which is primarely based on the blurry effect of an element and a background.",
```
Comment (CBID2) [inline database/frontend/ui-generators.json:130]: [Proofreading]
```suggestion
    "description": "Glassmorphism is a new design technique which is primarely based on the blurry effect of an element and a background.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @mohitbhusal11! :) I just made a suggestion on how to improve your description.

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm ! just commit the changes suggested by CBID2

**Review by CBID2 — APPROVED** [review]
LGTM!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**